src/components/Dropdown.module.scss

```diff
@@ -38,7 +38,6 @@
 .Button {
   @include button;
   background: var(--dark-purple);
-  width: 100%;
```

Comment:
This is actually necessary for the buttons in the dropdown. Otherwise they look like this:
<img width="245" alt="image" src="https://user-images.githubusercontent.com/96088385/171330031-e81df3e6-b5b5-4d8e-b4e8-bcd72fb02223.png">
But you can add another class to the non-dropdown buttons where you can over-ride this style if you need to. Alternatively, you could wrap the buttons in a div or something.